Sources/Networking/InternalAPI.swift

```diff
@@ -113,6 +113,21 @@ class InternalAPI {
         self.backendConfig.addDiagnosticsOperation(operation, delay: .long)
     }
 
+    @available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
+    func postAdEvents(events: [StoredEvent], completion: @escaping ResponseHandler) {
```

Comment:
The fact that this method already assumes the `StoredEvents` array contain `AdEvents` suggests me that we should probably refactor this to make it type safe (so that we can't possibly pass a `StoreEvent` that contains a `PaywallsEvent`, for example). Again, something to do on a separate PR